Lib/test/test_array.py

```diff
@@ -40,6 +40,11 @@ def test_bad_constructor(self):
         self.assertRaises(TypeError, array.array, 'xx')
         self.assertRaises(ValueError, array.array, 'x')
 
+    @support.cpython_only
+    def test_immutable(self):
```

Comment:
Please add a comment to explain the purpose of the test, something like:

"bpo-43908: array.array type is immutable"